benefits/core/migrations/0003_data_migration_order.py

```diff
@@ -0,0 +1,15 @@
+"""This migration simply ensures that data migrations (assumed to be files that start with 0002) will run sequentially.
+It does so by looking for all migration files starting with "0002" and declaring them in its dependency list.
+"""
+import os
+
+from django.conf import settings
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    migrations_directory = settings.BASE_DIR + "/benefits/core/migrations"
```

Comment:
See comment for [`settings.BASE_DIR`](https://github.com/cal-itp/benefits/blob/dev/benefits/settings.py#L12):

> Build paths inside the project like this: `os.path.join(BASE_DIR, ...)`

Reply:
Further, I think the `BASE_DIR` is `benefits/` (since that's where `settings.py` lives) - so you probably don't need that in the path parts.

Reply:
This is what I see in the Debug Console with a breakpoint at https://github.com/cal-itp/benefits/blob/dev/benefits/settings.py#L12:
```
> os.path.abspath(__file__)
'/home/calitp/app/benefits/settings.py'
> os.path.dirname(os.path.abspath(__file__))
'/home/calitp/app/benefits'
> os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
'/home/calitp/app'
```
For now, I will at least change `migrations_directory` to be built using `os.path.join`.